breezsdk/cmd/main.go

```diff
@@ -35,15 +32,13 @@ func main() {
 		log.Fatalf("failed to validate config %v", err)
 	}
 
-	creds, err := google.CredentialsFromJSON(context.Background(), []byte(os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")), "https://www.googleapis.com/auth/firebase.messaging")
```

Comment:
Can we rename the env variable to GOOLE_APPLICATION_CREDENTIALS_JSON so we can support both use cases?
In breez, for example, we don't use workload identity and we prefer not to save the credentials on the filesystem.

Reply:
This uses [ADC](https://cloud.google.com/docs/authentication/application-default-credentials) so you can pass  anything that [GOOGLE_APPLICATION_CREDENTIALS](https://cloud.google.com/docs/authentication/application-default-credentials#GAC) accepts. 

If you absolutely prefer to store a JSON on the filesystem rather than in an environment variable (or using workload identity), storing it in `$HOME/.config/gcloud/application_default_credentials.json` should work too.

Of course, you can also keep custom logic, but I would recommend following defaults.

Reply:
> If you absolutely prefer to store a JSON on the filesystem rather than in an environment variable (or using workload identity), storing it in `$HOME/.config/gcloud/application_default_credentials.json` should work too.

I actually wrote that we prefer NOT to save the credentials on the filesystem but rather use it as today which is a json string set in an environment variable. I don't see that [ADC](https://cloud.google.com/docs/authentication/application-default-credentials) supports that.

Reply:
Aha, my bad, I misunderstood. I think we can support that case indeed, one moment.

Reply:
@roeierez how's this?

Reply:
Looks good! 
@yaslama we just need to remember to change the env variable from `GOOGLE_APPLICATION_CREDENTIALS` to `GOOGLE_APPLICATION_CREDENTIALS_JSON` before deploying